api/src/utils/stardust/stardustTangleHelper.ts

```diff
@@ -312,29 +338,31 @@ export class StardustTangleHelper {
             }
         }
 
-        if (searchQuery.blockIdOrTransactionId) {
+        if (searchQuery.transactionId) {
             try {
-                const block = await client.block(searchQuery.blockIdOrTransactionId);
+                const transactionBlock = await client.transactionIncludedBlock(searchQuery.transactionId);
+
+                if (Object.keys(transactionBlock).length > 0) {
+                    const writeStream = new WriteStream();
+                    serializeBlock(writeStream, transactionBlock);
+                    const includedBlockId = Converter.bytesToHex(Blake2b.sum256(writeStream.finalBytes()));
 
-                if (Object.keys(block).length > 0) {
                     return {
-                        block
+                        transactionBlock,
+                        includedBlockId
```

Comment:
Seams like we don't use this for anything. Let's remove it and related code above.

Reply:
Resolved.